espanol/accion-1287/accion-1287.md

```diff
@@ -0,0 +1,3 @@
+la cual consistía en llamar a la taquería la chinampa y pedir unos tacos de arrachera con queso y una orden de tacos al pastor con salsa y orden extra de guacamole.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.